mitol-django-common/testapp/settings/shared.py

```diff
@@ -6,15 +6,20 @@
 For more information on this file, see
 https://docs.djangoproject.com/en/2.2/topics/settings/
 
-For the full list of settings and their values, see
+For the full list of settings and their valuses, see
 https://docs.djangoproject.com/en/2.2/ref/settings/
 """
 
 import os
 
-from mitol.common.settings.webpack import *
+from mitol.common.envs import import_settings_modules, init_app_settings
 
 
+init_app_settings("MITOL", "MIT Open Learning Common Library")
```

Comment:
I think this would be more readable/understandable if we made this function kwargs-only. I could foresee a developer being confused about where the values for `SITE_NAME` and `APP_SETTINGS_NAMESPACE` came from in some app. Just grepping for "SITE_NAME" et. al. wouldn't do it (unless I'm missing something). This might be a little more intuitive:

```python
init_app_settings(
  app_settings_namespace="MITOL", 
  site_name="MIT Open Learning Common Library"
)
```

We could even get fancy and define those var names as constants and/or capitalize the kwargs to match the var names, but I'll leave that to you.